hack/cozyreport.sh

```
@@ -134,6 +134,24 @@ kubectl get packagesources --no-headers | awk '$3 != "True"' | \
     kubectl describe packagesource $NAME > $DIR/describe.txt 2>&1
   done
 
+echo "Collecting source-watcher diagnostics..."
+DIR=$REPORT_DIR/kubernetes/source-watcher
+mkdir -p $DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)